### ISSUE TITLE
zcashd reports a misleading error message when zcash.conf is missing

### ISSUE THREAD

**Issue by bitcartel** [zcash/zcash#942]
zc.v0.11.2.z3

If zcash.conf does not exist, zcashd will report an incorrect error message:

```
EXCEPTION: St13runtime_error       
could not load param file at /home/simon/.zcash-params/z3-verification.key       
bitcoin in AppInit()       
```

The verification key does exist and was downloaded via fetch-params.sh.

**Comment by str4d** [comment]
`zcashd` can run without a `zcash.conf` fine. And you will find that the error message is correct - the param file shown in the message does not in fact exist.

`zcashd` expects the parameters to be stored in the `.zcash-params` directory with the same hierarchy as the regular `.zcash` folder. That is, the mainnet parameters are stored directly in that folder, while testnet and regtest parameters are stored in subfolders. Currently, `fetch-params.sh` downloads the parameters into the regtest subfolder and creates symlinks into the testnet subfolder, but does not create symlinks in the base folder. So what is happening here is that with no arguments, `zcashd` defaults to mainnet, and rightly can't find the parameters.

The question now is what to do about it. Throwing an error message for a missing `zcash.conf` is not the right solution, but we don't have another easy way to inform the user of their misconfiguration (which is what this is - with no `zcash.conf`, they should be calling `zcashd -testnet`). We could also just add symlinks for mainnet, but the mainnet parameters will not be the same as testnet at launch. And if we *do* add those symlinks, it's not actually helping the user to identify the real cause of the problem (the misconfiguration).

**Comment by daira** [comment]
Perhaps have ``fetch-params.sh`` create zero-length files where the mainnet keys should be. Then, if the code in zcashd that loads the params sees a zero-length file, it can say "You may be trying to load mainnet parameters, which do not exist yet. Run with 'testnet=1' in zcash.conf or use 'zcashd -testnet'."

**Comment by ebfull** [comment]
This will be addressed by #1312 (the parameter paths will not be dependent on the chain params).
